### ISSUE TITLE
How to Customize the plugin output such as footnote

### ISSUE THREAD

**Issue by MakerGYT** [hexojs/hexo-renderer-markdown-it#108]
@oncletom 
As [markdown-it-footnote doc](https://github.com/markdown-it/markdown-it-footnote#customize),
```js
md.renderer.rules.footnote_block_open = () => (
  '<h4 class="mt-3">Footnotes</h4>\n' +
  '<section class="footnotes">\n' +
  '<ol class="footnotes-list">\n'
);
```
That way, we can customize the output.
So, how to customize the footnote output in `hexo-renderer-markdown-it` after plug-in integration

**Comment by curbengh** [comment]
Available in v5.0.0+

``` js
hexo.extend.filter.register('markdown-it:renderer', function(md) {
  md.renderer.rules.footnote_block_open = () => (
    '<h4 class="mt-3">Footnotes</h4>\n' +
    '<section class="footnotes">\n' +
    '<ol class="footnotes-list">\n'
  );
});
```

Save it in a `.js` file in `scripts/` folder.

**Comment by MakerGYT** [comment]
Thanks
